src/components/ButtonWithConfirmDialog/ButtonWithConfirmDialog.tsx

```diff
@@ -3,13 +3,15 @@ import React from 'react';
 import {Button, Popover} from '@gravity-ui/uikit';
```

Comment:
There don't appear to be any new tests added for the new components and functionality. Adding unit and integration tests would help catch potential issues early. 

Cant insist but would be great

Reply:
Probably, I'll add tests later, but not in this PR - I want to finish functionality first